client-app/src/pages/account/views/account-invoices/index.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="mt-3">
+    <loading :active.sync="isLoading"></loading>
+    <invoices-filter :invoices-filter="invoicesList.listConfig.filters"
+                     :available-invoices-statuses="availableInvoicesStatuses"
+                     @filtersChanged="filtersChanged"></invoices-filter>
+    <div v-if="!isLoading" class="mt-3">
+      <p>{{ $t("account.invoices.grid.text-above") }}</p>
+      <b-table
+        id="users-table"
+        stacked="md"
+        striped
+        hover
+        :show-empty="true"
+        :empty-text="$t('account.invoices.no-invoices')"
+        :items="invoicesList.invoices"
+        :fields="invoicesList.listConfig.columns"
+        no-local-sorting
+        @sort-changed="sortChanged">
+        <template v-slot:cell(createdDate)="data">
+          <span>{{ data.value | moment("MMMM DD, YYYY") }}</span>
```

Comment:
We probably need `L`, `LL` or somehow get current format from vue or bootstrap-vue internationalization settings
https://momentjs.com/docs/#/displaying/format/ Localized formats

Reply:
done in last commit